backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -20,6 +21,8 @@ public class MqttEventHandler : EventHandlerBase
         private IRobotService RobotService =>
             _scopeFactory.CreateScope().ServiceProvider.GetRequiredService<IRobotService>();
 
+        private readonly Dictionary<string, System.Timers.Timer> _isarConnectionTimers = new();
```

Comment:
I think this belongs in its own EventHandler, it's not related to MQTT (receiving messages and emitting events) and should instead listen to the MQTT events and act accordingly but as a distinct service.
Maybe an "IsarConnectionMonitor" as its own EventHandler?

Reply:
Implemented